Woof/Woof/Model/Sitter/Sitter.swift

```diff
@@ -29,4 +29,62 @@ struct Sitter: Codable, Identifiable {
 
     /// The price per hour charged by the sitter.
     var pricePerHour: Double
+
+    /**
+     Creates a pet sitter instance with the specified parameters with the unique identifier.
+     Each instance of the new pet sitter has a unique id.
+
+     - Parameters:
+      - name: The first name of the pet sitter.
+      - surname: The last name of the pet sitter.
+      - phone: The phone number of the pet sitter.
+      - avatarURL: The URL that provides an avatar image of the pet sitter.
+      - bio: The additional information about the pet sitter.
+      - rating: The rating of the pet sitter.
+      - pricePerHour: The price per hour charged by the pet sitter.
+
+     */
+    init(
+        name: String,
+        surname: String,
+        phone: String,
+        avatarUrl: URL?,
+        bio: String,
+        rating: Double,
+        pricePerHour: Double
+    ) {
+        id = UUID()
```

Comment:
If we want to have a default value for a property, it's better to assign that when declare that property.
The ninth line should be: 
`    let id = UUID()`